README.md

```diff
@@ -108,23 +108,35 @@ If you are compiling on Linux, make sure to install the dependencies below.
 
 Additionally, make sure you have EGL installed. Since the package depends on your drivers, we cannot list it here.
 
+### Arch Linux
+
+```
+$ pacman -Sy grep gcc pkgconfig openssl alsa-lib cmake make python3 freetype2 awk libxcb
+```
+
 ### Debian/Ubuntu
 
 ```
-$ sudo apt install pkg-config libasound2-dev libssl-dev cmake libfreetype6-dev libexpat1-dev libxcb-composite0-dev
+$ apt-get install gcc pkg-config openssl libasound2-dev cmake build-essential python3 libfreetype6-dev libexpat1-dev libxcb-composite0-dev
```

Comment:
its more recommended to use apt than apt-get

Reply:
```suggestion
# apt install gcc pkg-config openssl libasound2-dev cmake build-essential python3 libfreetype6-dev libexpat1-dev libxcb-composite0-dev
```